### PR TITLE
[ID-8] core UI, alert dialog

### PR DESCRIPTION
**Pull request by hophongit98**
https://nartus.atlassian.net/browse/ID-8

### CODE REVIEW

File: packages/nartus_ui_package/lib/widgets/alert_dialog/nartus_alert_dialog.dart [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                textAlign: TextAlign.center,
+                maxLines: 4,
+                style: Theme.of(context)
+                    .textTheme
+                    .bodyMedium
+                    ?.copyWith(color: NartusColor.dark),
+              )),
+          Padding(
+              padding: const EdgeInsets.only(bottom: NartusDimens.padding10),
+              child: Column(crossAxisAlignment: CrossAxisAlignment.stretch, children: <Widget>[
```
Comment (suesitran) [inline packages/nartus_ui_package/lib/widgets/alert_dialog/nartus_alert_dialog.dart]: u dont need a second column here

**Review by suesitran — COMMENTED** [review]
just 1 improvement

File: packages/nartus_ui_package/lib/widgets/alert_dialog/nartus_alert_dialog.dart (hunk 1 above)
Comment (hophongit98) [reply packages/nartus_ui_package/lib/widgets/alert_dialog/nartus_alert_dialog.dart]: replaced with Sizebox. Is it okey?

File: packages/nartus_ui_package/lib/widgets/alert_dialog/nartus_alert_dialog.dart [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+      this.textButtonText,
+      this.onTextButtonSelected})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.fromLTRB(24, 40, 24, 40),
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
```
Comment (suesitran) [inline packages/nartus_ui_package/lib/widgets/alert_dialog/nartus_alert_dialog.dart:35]: u can use `crossAxisAlignment: CrossAxisAlignment.stretch` here, and remove the other column in line 90, or the SizedBox in line 83

**Review by suesitran — APPROVED** [review]

**Pull request merged**